proxy/http/HttpSM.cc

```diff
@@ -5460,9 +5460,14 @@ HttpSM::handle_http_server_open()
   //          server session's first transaction.
   if (nullptr != server_session) {
     NetVConnection *vc = server_session->get_netvc();
-    if (vc != nullptr && (vc->options.sockopt_flags != t_state.txn_conf->sock_option_flag_out ||
-                          vc->options.packet_mark != t_state.txn_conf->sock_packet_mark_out ||
-                          vc->options.packet_tos != t_state.txn_conf->sock_packet_tos_out)) {
+
+    SSLNetVConnection *ssl_vc = dynamic_cast<SSLNetVConnection *>(vc);
+    if (ssl_vc) {
+      ssl_vc->setClientVerifyEnable(t_state.txn_conf->ssl_client_verify_server);
+    }
+    if (vc != NULL && (vc->options.sockopt_flags != t_state.txn_conf->sock_option_flag_out ||
```

Comment:
I'm somewhat torn on this concept. We've done these sort of one-offs a few times now, and it's not really how the overridable configurations were intended to work. If we keep doing it like this, at some point, we'll have 10's of hundreds of these "copy" methods of moving configurations from the HttpSM into the various sub-systems.

I'd much prefer, if at all possible, that we consider the alternative of making the HttpSM's overridable configuration available to the subsystems.